src/SlippiGame.ts

```diff
@@ -69,13 +74,21 @@ export class SlippiGame {
       this.inputComputer,
       this.stockComputer,
     );
+
+    // Set up stadium stats calculation
+    this.stadiumStatsComputer = new Stats(opts);
+    this.stadiumStatsComputer.register(this.targetBreakComputer);
```

Comment:
Can we not have a second `stadiumStatsComputer`? I don't think we need it.
We can just register the `this.targetBreakComputer` with `this.statsComputer`.